templates/route.html

```diff
@@ -28,8 +28,8 @@
         {{#each this}}
         <div class="panel panel-default route">
             <a name="{{this.method}}" class="anchor"></a>
-            <div class="panel-heading h2 {{this.method}}">
-                <span class="method-title">
+            <div class="panel-heading {{this.method}}">
+                <span class="method-title h2">
```

Comment:
Moved h2 to prevent a regression when updating bootstrap

Before
![update](https://cloud.githubusercontent.com/assets/3107513/11877139/a4cc8a14-a4aa-11e5-995a-bb6b27ee5fe1.png)

After
![update-with-h2](https://cloud.githubusercontent.com/assets/3107513/11877148/b067eff8-a4aa-11e5-96d1-aa01a1aa9576.png)